Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
@MikeMcQuaid It's been proposed and implemented in less than three hours, based on an exchange between two maintainers that is buried in the code comment of a formula PR. That's why I chose “on a whim”. Maybe I'm asking for too much bureaucracy/communication here.

I'm not objecting a rename at all costs, but as I stated the benefit wasn't clear to me and the initial new name `patches-homebrew` was more confusing. For what it's worth, I did like the briefly used `brew-patches`, though that might carry the wrong message that these are patches for the `brew` repository (they are not). Of all the names suggested so far `formula-patches` is probably my favorite (no plural “e” for readability and I believe also grammar).

I'm still not entirely convinced that this is such a pressing issue, as few contributors have to work with the patches repository and (I believe) few are confused by it. A rename will probably just shift confusion from one group of people to another, at the expense of those who have already familiarized themselves with it and will be surprised to no longer find it in our repository list.